frontend/src/components/UserInfoCard.tsx

```diff
@@ -1,16 +1,162 @@
-import { UserInfo } from '../redux/user/types';
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import classNames from 'classnames';
+import { Trans } from 'react-i18next';
+import StarSharpIcon from '@material-ui/icons/StarSharp';
+import BuildSharpIcon from '@material-ui/icons/BuildSharp';
+import { BusinessIcon } from './RoleIcons';
+import { RoadmapRole, UserInfo } from '../redux/user/types';
 import { RoleType } from '../../../shared/types/customTypes';
+import css from './UserInfoCard.module.scss';
+import { EditableTextWithButtons } from './EditableText';
+import { table, TableRow } from './Table';
+import { sortKeyNumeric } from '../utils/SortUtils';
+import { idRoadmapSelector } from '../redux/roadmaps/selectors';
+import { StoreDispatchType } from '../redux';
+import { ModalTypes } from './modals/types';
+import { modalsActions } from '../redux/modals';
+import { api } from '../api/api';
 
-export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => (
-  <div>
-    <div>
-      {userInfo.email} ({userInfo.emailVerified ? 'verified' : 'not verified'})
-    </div>
-    {userInfo.roles.map((roadmapRole) => (
-      <div key={roadmapRole.roadmapId}>
-        {`Roadmap ${roadmapRole.roadmapId}: ${RoleType[roadmapRole.type]}`}
+const classes = classNames.bind(css);
+
+export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => {
+  const dispatch = useDispatch<StoreDispatchType>();
+  const [sent, setSent] = useState(false);
+  const [sending, setSending] = useState(false);
+
+  const ProjectRow: TableRow<RoadmapRole> = ({ item: roadmapRole, style }) => {
+    const { roadmapId, type } = roadmapRole;
+    const roadmap = useSelector(idRoadmapSelector(roadmapId));
+    return (
+      <div className={classes(css.virtualizedTableRow)} style={style}>
+        <td className="styledTd roleIcon">
```

Comment:
```suggestion
        <div className="styledTd roleIcon">
```

Noticed that this was complaining about incorrect nesting of elements, div should do fine here.